### PR TITLE
Release 1.9.1 doc changes

### PR DESCRIPTION
**Pull request by boyamurthy**
CSM 1.9.1 documentation changes . 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1109|

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by shanmydell — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

File: content/docs/csidriver/release/powerflex.md [hunk 1]
```diff
@@ -38,7 +40,7 @@ description: Release notes for PowerFlex CSI driver
...
 | If the volume limit is exhausted and there are pending pods and PVCs due to `exceed max volume count`, the pending PVCs will be bound to PVs and the pending pods will be scheduled to nodes when the driver pods are restarted. | It is advised not to have any pending pods or PVCs once the volume limit per node is exhausted on a CSI Driver. There is an open issue reported with kubenetes at https://github.com/kubernetes/kubernetes/issues/95911 with the same behavior. |
 | The PowerFlex Dockerfile is incorrectly labeling the version as 2.7.0 for the 2.8.0 version. | Describe the driver pod using ```kubectl describe pod $podname -n vxflexos``` to ensure v2.8.0 is installed. |
 | Standby controller pod is in crashloopbackoff state | Scale down the replica count of the controller pod's deployment to 1 using ```kubectl scale deployment <deployment_name> --replicas=1 -n <driver_namespace>``` |
```
Comment (adarsh-dell) [inline content/docs/csidriver/release/powerflex.md:46]: Hi @boyamurthy, I believe this should be removed from here, right?

Comment (adarsh-dell) [reply content/docs/csidriver/release/powerflex.md]: Same comment from rest of the driver's known issue section.

Comment (boyamurthy) [reply content/docs/csidriver/release/powerflex.md]: updated @adarsh-dell  , thanks .

**Pull request merged**